### PR TITLE
add security blurb to readme

### PR DESCRIPTION
**Pull request by gkulin**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -75,6 +71,8 @@ You can estimate the cost of this project's architecture with [Azure's pricing c
 
 TODO: team will add the guidelines here for best security practices.
 
+Each template has either [Managed Identity](https://learn.microsoft.com/en-us/entra/identity/managed-identities-azure-resources/overview) or Key Vault built in to eliminate the need for developers to manage these credentials. Applications can use managed identities to obtain Microsoft Entra tokens without having to manage any credentials. Additionally, we have added a [GitHub Action tool](https://github.com/microsoft/security-devops-action) that scans the infrastructure-as-code files and generates a report containing any detected issues. To ensure best practices in your repo we recommend anyone creating solutions based on our templates ensure that the [Github secret scanning](https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning) setting is enabled in your repos.
+
```
Comment (kristenwomack) [inline README.md:75]:
```suggestion
To be secure by design, we require templates in any Microsoft template collections to have the [Github secret scanning](https://docs.github.com/en/code-security/secret-scanning/about-secret-scanning) setting is enabled in your repos.'
```

Comment (kristenwomack) [reply README.md:75]: Do we want more firm requirements here?

**Review by kristenwomack — CHANGES_REQUESTED** [review]
Overall this is great, do we want to be firmer about security? I made a suggestion. And you could add, "we recommend this for your repos, as well?" - what do you think?

File: README.md (hunk 1 above)
Comment (gkulin) [reply README.md:75]: resolving after we discussed offline

**Review by kristenwomack — APPROVED** [review]
LGTM

**Pull request merged**